apps/condo/domains/notification/schema/SyncDeviceService.test.js

```diff
@@ -31,111 +19,111 @@ describe('SyncDeviceService', () => {
             const payload2 = getRandomTokenData({ pushToken: null }) // no pushToken, with meta
             const payload3 = getRandomTokenData() // with meta & pushToken
 
-            const [data] = await syncDeviceByTestClient(client, payload)
-
-            expect(data.id).not.toBeFalsy()
-            expect(data.deviceId).toEqual(payload.deviceId)
-            expect(data.pushTransport).toEqual(payload.pushTransport)
-            expect(data.pushToken).toBeNull()
-            expect(data.meta).toBeNull()
-            expect(data.owner).toBeNull()
-
-            const [data1] = await syncDeviceByTestClient(client, payload1)
-
-            expect(data1.id).not.toBeFalsy()
-            expect(data1.deviceId).toEqual(payload1.deviceId)
-            expect(data1.pushTransport).toEqual(payload1.pushTransport)
-            expect(data1.pushToken).toEqual(payload1.pushToken)
-            expect(data1.meta).toBeNull()
-            expect(data1.owner).toBeNull()
-
-            const [data2] = await syncDeviceByTestClient(client, payload2)
-
-            expect(data2.id).not.toBeFalsy()
-            expect(data2.deviceId).toEqual(payload2.deviceId)
-            expect(data2.pushTransport).toEqual(payload2.pushTransport)
-            expect(data2.pushToken).toBeNull()
-            expect(data2.meta).toEqual(payload2.meta)
-            expect(data2.owner).toBeNull()
-
-            const [data3] = await syncDeviceByTestClient(client, payload3)
-
-            expect(data3.id).not.toBeFalsy()
-            expect(data3.deviceId).toEqual(payload3.deviceId)
-            expect(data3.pushTransport).toEqual(payload3.pushTransport)
-            expect(data3.pushToken).toEqual(payload3.pushToken)
-            expect(data3.meta).toEqual(payload3.meta)
-            expect(data3.owner).toBeNull()
+            const [device] = await syncDeviceByTestClient(client, payload)
```

Comment:
How about test.each?
See examples in acquiring domain

Reply:
I'm not sure it fits here